lib/job-iteration/iteration.rb

```diff
@@ -112,15 +112,15 @@ def serialize # @private
       super.merge(
         "cursor_position" => cursor_position,
         "times_interrupted" => times_interrupted,
-        "total_time" => total_time,
+        "total_time" => total_time.to_f,
       )
     end
 
     def deserialize(job_data) # @private
       super
       self.cursor_position = job_data["cursor_position"]
       self.times_interrupted = job_data["times_interrupted"] || 0
-      self.total_time = job_data["total_time"] || 0
+      self.total_time = job_data["total_time"].to_f || 0.0
```

Comment:
Should we use `Float(...)` here, instead of `.to_f`? It has better error messages when it's fed things that are not floats. We do need to be careful with `nil` in that case:

```suggestion
      self.total_time = Float(job_data["total_time"] || 0.0)
```